src/toggle/toggle.styles.tsx

```diff
@@ -148,6 +158,16 @@ export const Container = styled.div<ContainerStyleProps>`
                         &:has(${HeaderContainer}:hover) {
                             background: ${Colour["bg-selected-hover"]};
                         }
+
+                        &:hover {
```

Comment:
this should also follow the `:has` selector like the background since the hover trigger behaviour is supposed to be the same. they can be merged together